packages/ses/src/evaluate.js

```diff
@@ -22,6 +22,7 @@ const { details: d } = assert;
  * @param {Array<Transform>} [options.localTransforms]
  * @param {Array<Transform>} [options.globalTransforms]
  * @param {bool} [options.sloppyGlobalsMode]
+ * @param setScopeProxy
```

Comment:
```suggestion
 * @param {WeakSet} knownScopeProxies
```

And ramifications thereof